datumaro/plugins/voc_format/format.py

```diff
@@ -171,6 +200,51 @@ def write_label_map(path, label_map):
 
             f.write('%s\n' % ':'.join([label_name, color_rgb, parts, actions]))
 
+def write_meta_file(path, label_map):
+    dataset_meta = {}
+
+    labels = {}
+    segmentation_colors = []
+    parts = {}
+    actions = {}
+
+    i = 0
+    for label_name, label_desc in label_map.items():
+        labels[str(i)] = label_name
+
+        segmentation_colors.append([int(label_desc[0][0]), int(label_desc[0][1]), int(label_desc[0][2])]
+            if label_desc[0] != None else None)
+
+        if not isinstance(label_desc, tuple) and \
+                label_desc[1] is not None and 0 < len(label_desc[1]):
```

Comment:
```suggestion
        if not isinstance(label_desc, tuple) and label_desc[1]:
```
`bool(container)` is equivalent to `container is not None and len(container) > 0`.

Shouldn't there be just `isinstance(label_desc, tuple)`?

Reply:
I removed this, it is useless for VOC.